pepper-apis/src/main/resources/db-changes/schema/study-invite-settings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200528-study-invite-setting-table">
+        <createTable tableName="study_invite_setting">
+            <column name="umbrella_study_id" type="bigint">
+                <constraints primaryKey="true" primaryKeyName="study_invite_setting_pk"
+                             references="umbrella_study(umbrella_study_id)" foreignKeyName="study_invite_setting_study_fk"
+                             deleteCascade="true"/>
+            </column>
+            <column name="invite_error_template_id" type="bigint">
+                <constraints nullable="true"
+                             references="template(template_id)"
+                             foreignKeyName="study_invite_setting_invite_error_tmpl_fk"/>
+            </column>
```

Comment:
Sure you want to dedicate a template to template foreign key? We expecting there will be more invite settings coming?

Reply:
Not sure what you mean by "template to template foreign key"?

Don't know if there's more invite settings coming, but I can imagine there being invite settings we might want: whether to check recaptcha or not, whether to do certain transformations to invite code before matching, etc.

Reply:
I meant "dedicate a table to just holding one foreign key". IMO these other possibilities as option sound unlikely, but just MO. Use your best judgement here.

Reply:
Well, if not this table, then I'm not sure what other table to put this.

Reply:
Maybe I can make this a more general "study setting" table, so we can throw other stuff in there in the future?

Reply:
Schema-wise, one logical candidate for this column would appear to be umbrella_study table as it would give you the correct cardinality. Creating a study_setting table might make sense too. Some of the columns in umbrella_study could be eventually migrated there. Both correct and map just fine to our object model.

Reply:
I'll start a `study_setting` table then!

Reply:
Had a hard time choosing between `setting` and `settings` but ultimately went with plural form since it's not just a single setting but rather will be a hodgepodge of settings. Is there a better word to capture the plurality nature of this?